src/FrontDoor/FrontDoor/Az.FrontDoor.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.5.0'
```

Comment:
You don't need to update version. Our tool will handle this. The only section you need to update in psd1 file is `CmdletsToExport` and `AliasesToExport`.